yts/test_suite_test.go

```diff
@@ -11,6 +13,29 @@ import (
 	"go.yaml.in/yaml/v4"
 )
 
+const (
+	ytsURL = "https://github.com/yaml/yaml-test-suite"
+	ytsTAG = "data-2022-01-17"
+	ytsDir = "testdata/" + ytsTAG
```

Comment:
filepath.Join should be used instead of strings concatenation

Reply:
impossible for a constant, just need to make it as a variable